pkg/cmd/workspace/code.go

```diff
@@ -204,3 +204,78 @@ func init() {
 	ideListStr := strings.Join(ids, ", ")
 	CodeCmd.Flags().StringVarP(&ideFlag, "ide", "i", "", fmt.Sprintf("Specify the IDE (%s)", ideListStr))
 }
+
+func StartIDE(ideId string, activeProfile config.Profile, workspaceId string, workspaceName string, projectName string, providerMetadata string, verbose bool) error {
```

Comment:
Please use a shorthand for arguments here (`workspaceId, workspaceName, projectName, providerMetadata string`).

Please rename the function to `StartAndOpenIDE`. `StartIDE` seems like we're starting an IDE.

Additionally, there's no need for `workspaceId` and `workspaceName` as arguments. They work interchangeably so the `workspaceId` is enough as an arg and name can be passed in its place if needed.

Reply:
> Please use a shorthand for arguments here (workspaceId, workspaceName, projectName, providerMetadata string).
> Please rename the function to StartAndOpenIDE. StartIDE seems like we're starting an IDE.

done

> Additionally, there's no need for workspaceId and workspaceName as arguments. They work interchangeably so the workspaceId is enough as an arg and name can be passed in its place if needed.

@Tpuljak  openIDE function only works with workspaceId, not workspaceName. we need to show an info message with the workspace name before opening the IDE. So, I added the workspace name as well

```go
views.RenderInfoMessage(fmt.Sprintf("Opening the project '%s' from workspace '%s' in %s", projectName, workspaceName, ideName))
```

Reply:
Ah, you're right.

Then I suggest the following:
- Since I misinterpreted what the function does in the begging, I suggest that you make the `OpenIDE` function public. They basically do the same thing. The `StartAndOpenIDE` is a bit bloated now. It does too many things which should be done (and are already done) by the caller.
- If the project name is not specified, we should always default to the first workspace project

```golang
	if verbose {
		ideList := config.GetIdeList()
		ideName := ""
		for _, ide := range ideList {
			if ide.Id == ideId {
				ideName = ide.Name
				break
			}
		}
		views.RenderInfoMessage(fmt.Sprintf("Opening the project '%s' from workspace '%s' in %s", projectName, workspaceName, ideName))
	}
```

I suggest that the code inside the `verbose` block be moved to a view utility which would take in `workspaceId, projectName, ideId string, ideList IDE[]` as arguments and print out the message.

This util can then also be reused in the Run method in `code.go`

Reply:
done 

 I have done the Run method changes in this PR: [#991](https://github.com/daytonaio/daytona/pull/991/files)